src/main/java/org/ibp/api/rest/labelprinting/GermplasmListLabelPrinting.java

```diff
@@ -281,78 +283,73 @@ void setMaxTotalResults(final int maxTotalResults) {
 	public LabelPrintingPresetDTO getDefaultSetting(final LabelsInfoInput labelsInfoInput, final String programUUID) {
 		final List<GermplasmListMeasurementVariableDTO> germplasmListMeasurementVariableDTOs =
 			this.germplasmListDataService.getGermplasmListDataTableHeader(labelsInfoInput.getListId(), programUUID);
-		final ArrayList<Integer> fields = new ArrayList<>();
+		final ArrayList<String> fields = new ArrayList<>();
 		germplasmListMeasurementVariableDTOs.stream().forEach(dto -> {
 			final TermId termId = TermId.getById(dto.getTermId());
 
 			switch (termId) {
 				case GID_ACTIVE_LOTS_COUNT:
-					fields.add(LabelPrintingStaticField.LOTS.getFieldId());
+					fields.add(FieldType.STATIC.getName() + LabelPrintingFieldUtils.UNDERSCORE + LabelPrintingStaticField.LOTS.getFieldId());
```

Comment:
@cuenyad Please move the combined key building to the Utils class, everywhere is BMSAPI

Reply:
done!